src/test/java/com/hocs/test/pages/private_office/PrivateOffice.java

```diff
@@ -23,6 +25,9 @@
     @FindBy(css = "label[for='PrivateOfficeDecision-CHANGE']")
     private WebElementFacade PrivateOfficeChangeMinisterRadioButton;
```

Comment:
PrivateOfficeChangeMinisterRadioButton
should start with a lower case
privateOfficeChangeMinisterRadioButton